harmony/aws.py

```diff
@@ -43,6 +46,25 @@ def aws_parameters(use_localstack, localstack_host, region):
         }
 
 
+def write_s3(txt, url):
```

Comment:
It's an ancient unix practice (and I think defacto standard for HTTP libraries) to put the destination as the first argument in a command/function that writes/copies something. So to avoid melting my brain would you mind switching the order of the arguments here?

Reply:
Certainly!

Reply:
Done.